pmg/views.py

```diff
@@ -210,6 +210,14 @@ def committee_detail(committee_id):
     """
     logger.debug("committee detail page called")
     committee = load_from_api('committee', committee_id)
+    YEAR_CUTOFF = 2012
```

Comment:
Yeah, I chose this arbitrarily--it was in line with the wireframe, but there's no particular reason why we'd stop there. We have data going back a very long time; I think I remember committee meetings dated to the mid-90's.